jax_triton/pallas/pallas_call.py

```diff
@@ -183,8 +231,8 @@ def pallas_call_lowering(ctx: mlir.LoweringRuleContext, *in_nodes, jaxpr, name,
       for out_shape in ctx.avals_out])
   i32_type = ir.IntegerType.get_signless(32)
   descriptor, keepalive = emit_triton_kernel_call(
-      ctx, name, asm, shared_mem, num_warps=num_warps, grid=grid,
-      metaparams=metaparams, dump_binary_path=None)
+      ctx, name, asm, shared_mem, num_warps=num_warps, grid=grid_spec.grid,
+      metaparams={}, dump_binary_path=None)
```

Comment:
Just curious: what are/were metaparams?

Reply:
Metaparams in Triton are things like BLOCK_SIZE (which are now passed to kernels via closure or partials).